pkg/archive/archive.go

```diff
@@ -633,6 +647,11 @@ func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error)
 				}
 
 				if err := ta.addTarFile(filePath, relFilePath); err != nil {
+					if err == errSkipFilePermissions {
+						// GH18120. Note next line is deliberately NOT logrus or %q
+						fmt.Printf("Skipping '%s' - Permission was denied\n", filePath)
```

Comment:
Because I want it directly in the client output. Let me repro this evening and post up the difference and it will hopefully be obvious.